_posts/2018-11-06-mongodb-transactions.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+authors: [jan_van_der_veken]
+title: "Transactions in MongoDB 4.0"
+image: /img/2018-11-08-mongodb-europe-2018/main-image-mdbe.png
+tags: [Development,MongoDB,DBA,Data,Transactions,ACID,Conference]
+category: Development
+comments: true
+---
+
+# Table of Contents
+1. [Introduction](#introduction)
+2. [Relational vs. document database](#Relational-vs.-document-database)
+3. [When to use transactions?](#when-to-use-transactions)
+4. [Technical details](#technical-details)
+5. [Practical](#practical)
+6. [Best Practices](#best-practices)
+5. [Conclusion](#conclusion)
+
+## Introduction
+- [How and When to Use Multi-Document Distributed Transactions, Aly Cabral](https://mongodbeurope2018.sched.com/event/FmAR/how-and-when-to-use-multi-document-distributed-transactions)
+- [MongoDB: Building a New Transactional Model, Keith Bostic](https://mongodbeurope2018.sched.com/event/FmAW/mongodb-building-a-new-transactional-model)
+
+The sessions I was looking forward to the most at MongoDB Europe 2018 were the two sessions about multi-document transactions, the most talked about feature of the MongoDB 4.0 release.
+In the morning I attended a session by Aly Cabral which was very practically oriented.
+In the afternoon there was a more esoteric yet very interesting session by Keith Bostic who provided some insight into the inner workings of the WiredTiger storage engine and the difficulties the MongoDB team had to overcome to implement the new transaction model.
+
+To give some background, as a longtime Oracle DBA, I always found it odd that a database would lack what I had always considered a crucial database feature during my Oracle days, so I was naturally curious to know more about MongoDB's implementation and how it would compare to a typical relational database.
```

Comment:
```suggestion
To give some background, as a longtime Oracle DBA, I always found it odd that a database would lack what I had always considered a crucial database feature during my Oracle days.
So I was naturally curious to know more about MongoDB's implementation and how it would compare to a typical relational database.
```

Reply:
Het is af te raden een zin met "So" te beginnen, grammaticaal is er niks mis met een samengestelde zin.